### ISSUE TITLE
Multitenancy: Issue executing DB request

### ISSUE THREAD

**Issue by julsemaan** [inverse-inc/packetfence#2743]
Seems to be happening when receiving a DHCP packet:

```
Oct 31 11:28:29 pf-julien pfqueue: pfqueue(6647) ERROR: [mac:94:db:c9:38:85:5b] Database query failed with non retryable error: Unknown column 'node.tenant_id' in 'where clause' (errno: 1054) [SELECT `locationlog`.`switch` AS `last_switch`, `locationlog`.`port` AS `last_port`, `locationlog`.`vlan` AS `last_vlan`, IF(ISNULL(`locationlog`.`connection_type`), '', `locationlog`.`connection_type`) AS `last_connection_type`, IF(ISNULL(`locationlog`.`connection_sub_type`), '', `locationlog`.`connection_sub_type`) AS `last_connection_sub_type`, `locationlog`.`dot1x_username` AS `last_dot1x_username`, `locationlog`.`ssid` AS `last_ssid`, `locationlog`.`stripped_user_name` AS `stripped_user_name`, `locationlog`.`realm` AS `realm`, `locationlog`.`switch_mac` AS `last_switch_mac`, `locationlog`.`start_time` AS `last_start_time`, `locationlog`.`role` AS `last_role`, UNIX_TIMESTAMP(`locationlog`.`start_time`) AS `last_start_timestamp` FROM `locationlog` WHERE ( ( ( `end_time` = ? AND `mac` = ? ) AND `node`.`tenant_id` = ? ) )]{0000-00-00 00:00:00, 94:db:c9:38:85:5b, 1} (pf::dal::db_execute)
Oct 31 11:28:29 pf-julien pfqueue: pfqueue(6647) INFO: [mac:94:db:c9:38:85:5b] Instantiate profile default (pf::Connection::ProfileFactory::_from_profile)
Oct 31 11:28:29 pf-julien pfqueue: pfqueue(6649) ERROR: [mac:94:db:c9:38:85:5b] Database query failed with non retryable error: Unknown column 'node.tenant_id' in 'where clause' (errno: 1054) [SELECT `locationlog`.`switch` AS `last_switch`, `locationlog`.`port` AS `last_port`, `locationlog`.`vlan` AS `last_vlan`, IF(ISNULL(`locationlog`.`connection_type`), '', `locationlog`.`connection_type`) AS `last_connection_type`, IF(ISNULL(`locationlog`.`connection_sub_type`), '', `locationlog`.`connection_sub_type`) AS `last_connection_sub_type`, `locationlog`.`dot1x_username` AS `last_dot1x_username`, `locationlog`.`ssid` AS `last_ssid`, `locationlog`.`stripped_user_name` AS `stripped_user_name`, `locationlog`.`realm` AS `realm`, `locationlog`.`switch_mac` AS `last_switch_mac`, `locationlog`.`start_time` AS `last_start_time`, `locationlog`.`role` AS `last_role`, UNIX_TIMESTAMP(`locationlog`.`start_time`) AS `last_start_timestamp` FROM `locationlog` WHERE ( ( ( `end_time` = ? AND `mac` = ? ) AND `node`.`tenant_id` = ? ) )]{0000-00-00 00:00:00, 94:db:c9:38:85:5b, 1} (pf::dal::db_execute)
Oct 31 11:28:29 pf-julien pfqueue: pfqueue(6649) INFO: [mac:94:db:c9:38:85:5b] Instantiate profile default (pf::Connection::ProfileFactory::_from_profile)
```

The issue seems to be that the node table isn't joined in that query so there is no point into filtering for the tenant_id of the node

**Comment by jrouzierinverse** [comment]
Fixed by b3c412da0ad0bbdb04dc922bed426752ea3b53c5

**Comment by extrafu** [comment]
@julsemaan Should be closed?
